.storybook/main.ts

```diff
@@ -0,0 +1,24 @@
+import type { StorybookConfig } from "@storybook/nextjs";
+
+const config: StorybookConfig = {
+  stories: [
+    "../components/**/*.stories.mdx",
```

Comment:
Why should Mdx stories be added?

Reply:
I added it for future use cases, if we needed MDX story documentation. But I think we should remove it for now since it's currently not needed.